orchestrator.py

```
@@ -37,334 +38,371 @@ class OrchestrationEngine:
         return results
-    
+
     async def execute_protocol(self, name: str, inputs: Dict) -> Any:
         """Execute a protocol"""
         from protocols.loader import load_protocol
+
         protocol = load_protocol(name)
         if protocol:
-            return protocol['task']()
+            return protocol["task"]()
         raise Exception(f"Protocol {name} not found")
```

Ruff (0.11.9)

Create your own exception

(TRY002)